src/memory/combat_manager.rs

```diff
@@ -410,6 +413,13 @@ impl UnityItem for CombatPlayer {
         let current_hp = process.read_pointer_path::<u32>(item_ptr, &[0x180, 0x28, 0x58])?;
         let current_mp = process.read_pointer_path::<u32>(item_ptr, &[0x180, 0x30, 0x58])?;
 
+        let base_hp = process.read_pointer_path::<u32>(item_ptr, &[0x150, 0x30, 0x20])?;
+        let base_mp = process.read_pointer_path::<u32>(item_ptr, &[0x150, 0x30, 0x50])?;
+
+        // this get the current level up upgrades + 1, so if they're level 2 they should
```

Comment:
I chose this way because i had no way to reach into PlayerPartyManager or CharacterStatsManager from here - it works well enough so unless it gives us issues - lets just see how it goes. :)